requirements.txt

```diff
@@ -1,6 +1,7 @@
 oauthlib
 PyJWT
 neptune-cli
+neptune-lib==0.1.0
```

Comment:
Caught my eye: you probably want 0.1.1 :)

Reply:
Thanks :+1: